### PR TITLE
fix(query): adjusted severity rating and added searchLine in rbac_wildcard_in_rule k8s rule

### PR DESCRIPTION
**Pull request by Churro**
**Proposed Changes**

- Add `searchLine` to point to specific rule attributes
- Combine the CxPolicy rules into one
- Extend description
- Adjust rating from LOW to MEDIUM (or HIGH?) for the following reasons:
  - The rule [rbac_roles_with_read_secrets_permissions](https://github.com/Checkmarx/kics/blob/master/assets/queries/k8s/rbac_roles_with_read_secrets_permissions/query.rego) checks for a specific "permission subset" of this rule and  is rated MEDIUM. This rule here implicitly also covers the same case as the other rule by checking for wildcards, which allow even wider access rights
  - Wildcard permissions on resources (pods, depoyments, secrets, ...) also include create, update, patch, delete, bind, escalate, impersonate verbs. Attackers may abuse this to perform mutating changes, e.g., create malicious pods, delete secrets, change role bindings, impersonate users/groups/service accounts. Overall, this can put the availability of deployments at risk, in addition to violating confidentiality and integrity
  - The worst-case setting would be * on all `apiGroups`, `resources`, and `verbs`, essentially meaning cluster admin rights and this would even justify a HIGH severity rating. Depending on what is set to *, the potential blast radius varies. If you think it would be reasonable to assume the worst-case scenario here, HIGH would be appropriate - otherwise at least MEDIUM, I'd say 😊

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

**Review by rafaela-soares — COMMENTED** [review]
Amazing refactor and improvement, @Churro! Thank you so much!

LGTM! I will only wait for the severity approval.

**Review by joaorufi — DISMISSED** [review]

**Comment by rafaela-soares** [comment]
Hello, @Churro! Can you update the severity to HIGH, please?

**Comment by Churro** [comment]
Hi @rafaela-soares, done :smiley:

**Review by rafaela-soares — APPROVED** [review]
LGTM! 🚀 

Thank you so much, @Churro!

**Pull request merged**
